src/DonationForms/Blocks/DonationFormBlock/resources/editor/Edit.tsx

```diff
@@ -29,146 +28,55 @@ export default function Edit({clientId, attributes, setAttributes}: BlockEditPro
-                    <PanelRow>
-                        <SelectControl
-                            label={__('Form Format', 'give')}
-                            value={formFormat}
-                            options={[
-                                {
-                                    label: __('Full Form', 'give'),
-                                    value: 'full',
-                                },
-                                {
-                                    label: __('Reveal', 'give'),
-                                    value: 'reveal',
-                                },
-                                {
-                                    label: __('Modal', 'give'),
-                                    value: 'modal',
-                                },
-                            ]}
-                            onChange={(value) => {
-                                setAttributes({formFormat: value});
-                            }}
-                        />
-                    </PanelRow>
-                    {showOpenFormButton && (
-                        <PanelRow>
-                            <TextControl
-                                label={__('Open Form Button', 'give')}
-                                value={openFormButton}
-                                onChange={(value) => {
-                                    setAttributes({openFormButton: value});
-                                }}
-                            />
-                        </PanelRow>
-                    )}
-                    <PanelRow>
-                        {formId && (
-                            <ExternalLink
-                                href={`/wp-admin/edit.php?post_type=give_forms&page=givewp-form-builder&donationFormID=${formId}`}
-                            >
-                                {__('Edit donation form', 'give')}
-                            </ExternalLink>
-                        )}
-                    </PanelRow>
-                </PanelBody>
-            </InspectorControls>
+    const showOpenFormButton = formFormat === 'newTab' || formFormat === 'modal';
+    const isv2Form = forms && id && isTemplateForm(forms, id);
+    const isv3Form = forms && id && !isTemplateForm(forms, id) && !isLegacyForm(forms, id);
 
-            {/*block preview*/}
-            <div {...useBlockProps()}>
-                {formId && showPreview ? (
-                    <BlockPreview
-                        clientId={clientId}
-                        formId={formId}
-                        formFormat={formFormat}
-                        openFormButton={openFormButton}
-                    />
-                ) : (
-                    <div className="givewp-form-block--container">
-                        <Logo />
+    if (isv2Form) {
+        return (
+            <div className={!!isSelected ? `${className} isSelected` : className}>
+                <Inspector {...{...props}} />
```

Comment:
Let's be consistent here, right now there are two different UIs for selecting the form. Can we remove the old Inspector, pull out the legacy form controls, and put everything into `DonationFormBlockControls`?

Reply:
Resolved: abf51e18deb49c99dfedafaa59816943289fe8f7 https://github.com/impress-org/givewp/pull/7082/commits/f51ed8228d283589f13ff4e4dc4573b57ddd2b10